### PR TITLE
Search movies

### PR DESCRIPTION
**Pull request by alienalien13**

### CODE REVIEW

File: src/schema/graphql.d.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+declare module '*.graphql' {
+  import {DocumentNode} from 'graphql';
+
+  const value: DocumentNode;
+  export default value;
+}
```
Comment (maybelinot) [inline src/schema/graphql.d.ts]: what does this file do? Don't see any import of it.. LGTM except for this on

**Comment by flashlee** [comment]
@maybelinot @alienalien13 Guys, didn't you already merged these changes ? ))
https://github.com/illWatchers/ill-watch/pull/33

File: src/schema/graphql.d.ts (hunk 1 above)
Comment (flashlee) [reply src/schema/graphql.d.ts]: @maybelinot +
@alienalien13 can you give us some clue on that? )

Comment (alienalien13) [reply src/schema/graphql.d.ts]: This is redundant :) I'll remove it

**Comment by maybelinot** [comment]
@flashlee weird, rebase&merge created new commits, wtf
@alienalien13 rebase master pls

**Pull request merged**